applications/luci-app-usteer/htdocs/luci-static/resources/view/usteer/usteer.js

```diff
@@ -144,22 +169,55 @@ function collectWlanAPInfos(compactconnectioninfo_table_entries, wlanAPInfos) {
 	}
 };
 
+var callNetworkRrdnsLookup = rpc.declare({
+	object: 'network.rrdns',
+	method: 'lookup',
+	params: [ 'addrs', 'timeout', 'limit' ],
+	expect: { '': {} }
+});
+
+
 function collectRemoteHosts (remotehosttableentries,Remotehosts) {
+	var ipAddrs = [];
 	for (var IPaddr in Remotehosts) {
```

Comment:
Replace this for loop using a filter:

```js
const getUndefinedDnsCacheIPs = (Remotehosts, dns_cache) =>
  Object.keys(Remotehosts).filter(IPaddr => !dns_cache.hasOwnProperty(IPaddr));
ipAddrs = getUndefinedDnsCacheIPs(Remotehosts, dns_cache);
```
(I'm freestyling now so it'll need testing)

Reply:
nice freestyle, worked without editing :)

Reply:
:D